openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/DatabaseRepository.java

```diff
@@ -105,6 +106,18 @@ public EntityInterface getParentEntity(Database entity, String fields) {
     return Entity.getEntity(entity.getService(), fields, Include.ALL);
   }
 
+  @Override
+  public void entityRelationshipReindex(Database original, Database updated) {
+    super.entityRelationshipReindex(original, updated);
+
+    // Update search indexes of assets and entity on database displayName change
+    if (!Objects.equals(original.getDisplayName(), updated.getDisplayName())) {
```

Comment:
Do we need to do this explicitly, will the updateEntity() in search not take care of this ?

Reply:
updateEntity() currents only updates the inherited and propagated fields in children and not other fields of the parent entity